.github/workflows/continuous_integration.yml

```diff
@@ -1,10 +1,9 @@
 name: Continuous Integration
 
 on:
-  pull_request:
   push:
     branches:
-      - dev
+      - '**'
```

Comment:
same comment as the other PR. You can just use `on: push` here, to trigger on all branches. See [here](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-events-to-trigger-workflows) for details.